src/Time/Date.elm

```diff
@@ -29,27 +29,32 @@ represent any date of the proleptic Gregorian calendar.
 
 # Dates
 
-@docs Date, date, year, month, day, Weekday, weekday
+@docs Date
 
 
-# Manipulating Dates
+# Constructing Dates
```

Comment:
Actually, **Manipulating** is more accurate; the identified functions don't create any from scratch.